pkg/auth/dependency/sso/utils.go

```diff
@@ -60,7 +99,7 @@ func GetScope(scope Scope, defaultScope Scope) Scope {
 // RedirectURI generates redirect uri from URLPrefix and provider name
 func RedirectURI(URLPrefix string, providerName string) string {
 	u, _ := url.Parse(URLPrefix)
-	path := fmt.Sprintf("/sso/%s/auth_handler", providerName)
+	path := fmt.Sprintf("/auth/sso/%s/auth_handler", providerName)
```

Comment:
Auth gear should not need to have knowledge of the `/auth` prefix? If developer use the auth gear without auth gear, it should work. When there is the gateway, we should config the `URLPrefix` to be `http://api-domain/auth`? Does it work?